torch/_tensor_str.py

```diff
@@ -266,6 +266,20 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        suffixes.append('dtype=' + str(self.dtype))
+        suffixes.append('quantization_scheme=' + 'per_tensor_affine')
```

Comment:
there's no way other place to get the quantization scheme currently?  Like as an enum or something (where we could have another function that maps them to strings)?  Putting this in the print function doesn't seem good.

Reply:
Yeah, we'll change that when we add q_scheme(), I'll add a comment